watcher/src/bin/main.rs

```diff
@@ -7,21 +7,26 @@
 //! blocks are synced.
 
 use mc_watcher::{
-    config::WatcherConfig, watcher::Watcher, watcher_db::create_or_open_rw_watcher_db,
+    config::WatcherConfig, verification_reports_collector::VerificationReportsCollector,
+    watcher::Watcher, watcher_db::create_or_open_rw_watcher_db,
 };
 
-use mc_common::logger::{create_app_logger, o};
+use mc_common::logger::{create_app_logger, log, o};
 use mc_ledger_sync::ReqwestTransactionsFetcher;
+use std::{thread::sleep, time::Duration};
 use structopt::StructOpt;
 
+const SYNC_RETRY_INTERVAL: Duration = Duration::from_secs(1);
```

Comment:
Why is this set here as opposed to either from config, or via using something like the retry crate?

Reply:
Its there because realistically no-one will ever choose a different value. `retry` is inappropriate since this is actually a polling interval so the name is misleading. I'll add it to the command line config.